json-endpoints/class.wpcom-json-api-site-settings-v1-3-endpoint.php

```diff
@@ -0,0 +1,159 @@
+		'comment_whitelist'                    => '(bool) Moderate comments unless author has a previously-approved comment?',
+		'comment_max_links'                    => '(int) Moderate comments that contain X or more links',
+		'moderation_keys'                      => '(string) Words or phrases that trigger comment moderation, one per line',
+		'blacklist_keys'                       => '(string) Words or phrases that mark comment spam, one per line',
+		'lang_id'                              => '(int) ID for language blog is written in',
+		'locale'                               => '(string) locale code for language blog is written in',
+		'wga'                                  => '(array) Google Analytics Settings',
+		'disabled_likes'                       => '(bool) Are likes globally disabled (they can still be turned on per post)?',
+		'disabled_reblogs'                     => '(bool) Are reblogs disabled on posts?',
+		'jetpack_comment_likes_enabled'        => '(bool) Are comment likes enabled for all comments?',
+		'sharing_button_style'                 => '(string) Style to use for sharing buttons (icon-text, icon, text, or official)',
+		'sharing_label'                        => '(string) Label to use for sharing buttons, e.g. "Share this:"',
+		'sharing_show'                         => '(string|array:string) Post type or array of types where sharing buttons are to be displayed',
+		'sharing_open_links'                   => '(string) Link target for sharing buttons (same or new)',
+		'twitter_via'                          => '(string) Twitter username to include in tweets when people share using the Twitter button',
+		'jetpack-twitter-cards-site-tag'       => '(string) The Twitter username of the owner of the site\'s domain.',
+		'eventbrite_api_token'                 => '(int) The Keyring token ID for an Eventbrite token to associate with the site',
+		'holidaysnow'                          => '(bool) Enable snowfall on front end of site?',
+		'timezone_string'                      => '(string) PHP-compatible timezone string like \'UTC-5\'',
+		'gmt_offset'                           => '(int) Site offset from UTC in hours',
+		'date_format'                          => '(string) PHP Date-compatible date format',
+		'time_format'                          => '(string) PHP Date-compatible time format',
+		'start_of_week'                        => '(int) Starting day of week (0 = Sunday, 6 = Saturday)',
+		'jetpack_testimonial'                  => '(bool) Whether testimonial custom post type is enabled for the site',
+		'jetpack_testimonial_posts_per_page'   => '(int) Number of testimonials to show per page',
+		'jetpack_portfolio'                    => '(bool) Whether portfolio custom post type is enabled for the site',
+		'jetpack_portfolio_posts_per_page'     => '(int) Number of portfolio projects to show per page',
+		Jetpack_SEO_Utils::FRONT_PAGE_META_OPTION => '(string) The SEO meta description for the site.',
```

Comment:
I worry about using `Jetpack_SEO_Utils::FRONT_PAGE_META_OPTION` here, as if the seo-tools module is deactivated the class won't be available and this would kick an error.  Can we just hardcode the option name?

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/7869/commits/e6aed25d9eb93e51e85d0ff9d589584f08ea7cb6

Reply:
@georgestephanis @allendav I worked around this issue by including required SEO files in `module-extras.php`, so they will be loaded even if the module is deactivated:
https://github.com/Automattic/jetpack/blob/master/modules/module-extras.php#L23

 If that is the only concern we can continue using the constants. I also don't think it would cause any issues if you switch to using strings instead.